moi.gov.sa,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
moj.go.jp,"HTTPSConnectionPool(host='moj.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7bbfbc7c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='moj.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bb9fd2b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
moj.gov.sa,"HTTPSConnectionPool(host='moj.gov.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='moj.gov.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
moja-delatnost.rs,"HTTPSConnectionPool(host='moja-delatnost.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='moja-delatnost.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
mojakompanija.com,"HTTPSConnectionPool(host='www.mojakompanija.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mojakompanija.com', port=443): Read timed out. (read timeout=2)"
mojaljekarna.hr,"HTTPSConnectionPool(host='mojaljekarna.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mojaljekarna.hr', port=443): Read timed out. (read timeout=2)"
moje-podjetje.net,"HTTPSConnectionPool(host='www.moje-podjetje.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.moje-podjetje.net', port=443): Read timed out. (read timeout=2)"
mojevrijeme.hr,"HTTPSConnectionPool(host='mojevrijeme.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mojevrijeme.hr', port=443): Max retries exceeded with url: /magazin/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fd7b9d492e0>, 'Connection to www.mojevrijeme.hr timed out. (connect timeout=2)'))"
momoshop.com.tw,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
mondaq.com,"HTTPSConnectionPool(host='mondaq.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b82dc760>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mondaq.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bb21f670>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mondovolantino.it,"HTTPSConnectionPool(host='mondovolantino.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mondovolantino.it' doesn't match 'www.mondovolantino.it'"")))","HTTPSConnectionPool(host='mondovolantino.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'mondovolantino.it' doesn't match 'www.mondovolantino.it'"")))"
montreal.qc.ca,"HTTPSConnectionPool(host='montreal.qc.ca', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b7968bb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='montreal.qc.ca', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b9100670>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mopt.go.cr,"HTTPSConnectionPool(host='mopt.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b7c8d6a0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='mopt.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b9b95a90>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
morelos.gob.mx,"HTTPSConnectionPool(host='morelos.gob.mx', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='morelos.gob.mx', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
mosquitoproject.org,"HTTPSConnectionPool(host='mosquitoproject.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='mosquitoproject.org', port=80): Read timed out. (read timeout=2)"
motorguia.net,"HTTPSConnectionPool(host='motorguia.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='motorguia.net', port=443): Read timed out. (read timeout=2)"
mots.go.th,"HTTPSConnectionPool(host='mots.go.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mots.go.th', port=443): Read timed out. (read timeout=2)"
movistar.co,"HTTPSConnectionPool(host='www.movistar.co', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.movistar.co', port=443): Read timed out. (read timeout=2)"
movistar.com.ar,"HTTPSConnectionPool(host='movistar.com.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='movistar.com.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
movistarplus.es,"HTTPSConnectionPool(host='movistarplus.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'movistarplus.es' doesn't match either of '*.dof6.com', 'dof6.com'"")))","HTTPSConnectionPool(host='www.movistar.es', port=443): Read timed out. (read timeout=2)"
mozaweb.com,"HTTPSConnectionPool(host='www.mozaweb.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mozaweb.com', port=443): Read timed out. (read timeout=2)"
mozaweb.hu,"HTTPSConnectionPool(host='www.mozaweb.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mozaweb.hu', port=443): Read timed out. (read timeout=2)"
mp3kurt.net,"HTTPSConnectionPool(host='mp3kurt.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1131)')))","HTTPConnectionPool(host='mp3kurt.net', port=80): Read timed out. (read timeout=2)"
mp3party.net,"HTTPSConnectionPool(host='mp3party.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mp3party.net', port=443): Read timed out. (read timeout=2)"
mrdiy.com.my,"HTTPSConnectionPool(host='www.mrdiy.com.my', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mrdiy.com.my', port=443): Read timed out. (read timeout=2)"
mrsbrose.com,"HTTPConnectionPool(host='www.mrsbrose.com', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.mrsbrose.com', port=80): Read timed out. (read timeout=2)"
ms.ro,"HTTPSConnectionPool(host='ms.ro', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7bc1f1340>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='ms.ro', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bbc59fd0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
mscbs.gob.es,"HTTPSConnectionPool(host='mscbs.gob.es', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b84158e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mscbs.gob.es', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b8043e50>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
msp.gob.ec,"HTTPSConnectionPool(host='msp.gob.ec', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7bac06490>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='msp.gob.ec', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7ba5539d0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
mspas.gob.gt,"HTTPSConnectionPool(host='mspas.gob.gt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='mspas.gob.gt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
msudrf.ru,"HTTPSConnectionPool(host='msudrf.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPConnectionPool(host='msudrf.ru', port=80): Read timed out. (read timeout=2)"
mt.gov.vn,"HTTPSConnectionPool(host='mt.gov.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mt.gov.vn', port=443): Read timed out. (read timeout=2)"
mtoys.co.il,"HTTPSConnectionPool(host='mtoys.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mtoys.co.il', port=443): Read timed out. (read timeout=2)"
mtroos.com,"HTTPSConnectionPool(host='mtroos.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='mtroos.com', port=443): Read timed out. (read timeout=2)"
mufg.jp,"HTTPSConnectionPool(host='mufg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b7111f10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mufg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b7ee2940>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
multicoisas.com.br,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
mumsnet.com,"HTTPSConnectionPool(host='www.mumsnet.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.mumsnet.com', port=443): Read timed out. (read timeout=2)"
municipiosefreguesias.pt,"HTTPSConnectionPool(host='municipiosefreguesias.pt', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='municipiosefreguesias.pt', port=80): Read timed out. (read timeout=2)"
murciasalud.es,"HTTPSConnectionPool(host='murciasalud.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'murciasalud.es' doesn't match 'www.murciasalud.es'"")))","HTTPConnectionPool(host='www.murciasalud.es', port=80): Read timed out. (read timeout=2)"
musor.tv,Exceeded 30 redirects.,Exceeded 30 redirects.
muzika.hr,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
muzikbahcesi.com,"HTTPSConnectionPool(host='muzikbahcesi.com', port=443): Read timed out.","HTTPSConnectionPool(host='muzikbahcesi.com', port=443): Read timed out."
mvb.lt,"HTTPSConnectionPool(host='mvb.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7bb190b80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mvb.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bac42ca0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mvep.hr,"HTTPSConnectionPool(host='mvep.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7ba8e2370>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mvep.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b34e5eb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
mydns.jp,"HTTPSConnectionPool(host='mydns.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7ba534550>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='mydns.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bbab9580>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
myfanbase.de,"HTTPSConnectionPool(host='www.myfanbase.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.myfanbase.de', port=443): Read timed out. (read timeout=2)"
myntra.com,"HTTPSConnectionPool(host='www.myntra.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.myntra.com', port=443): Read timed out. (read timeout=2)"
myq-see.com,"HTTPSConnectionPool(host='myq-see.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b942fd90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='myq-see.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b20b4dc0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
myshop.lk,"HTTPSConnectionPool(host='www.myshop.lk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.myshop.lk', port=443): Read timed out. (read timeout=2)"
mysound.jp,"HTTPSConnectionPool(host='mysound.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='mysound.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
n-sport.net,"HTTPSConnectionPool(host='n-sport.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b79c8a00>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='n-sport.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b8b26970>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
naftemporiki.gr,"HTTPSConnectionPool(host='naftemporiki.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='naftemporiki.gr', port=443): Read timed out. (read timeout=2)"
nagano.lg.jp,"HTTPSConnectionPool(host='nagano.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b9980700>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='nagano.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bb0994c0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
nagoya-u.ac.jp,"HTTPSConnectionPool(host='nagoya-u.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7bbf22f10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='nagoya-u.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b3c9f910>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
nahdionline.com,"HTTPSConnectionPool(host='www.nahdionline.com', port=443): Read timed out.","HTTPSConnectionPool(host='www.nahdionline.com', port=443): Read timed out."
nairobiwire.com,"HTTPSConnectionPool(host='nairobiwire.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nairobiwire.com', port=80): Read timed out. (read timeout=2)"
nam.fi,"HTTPSConnectionPool(host='nam.fi', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b942fd30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='nam.fi', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b5cf6310>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
naoko.asia,"HTTPSConnectionPool(host='www.naoko.asia', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.naoko.asia', port=443): Read timed out. (read timeout=2)"
naoussa.be,"HTTPSConnectionPool(host='naoussa.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b0891640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='naoussa.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7bb4d8c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nasdaq.com,"HTTPSConnectionPool(host='www.nasdaq.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.nasdaq.com', port=80): Read timed out. (read timeout=2)"
nasezdravie.sk,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
nationalarchives.gov.uk,"HTTPSConnectionPool(host='nationalarchives.gov.uk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='nationalarchives.gov.uk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
nationstrust.com,"HTTPSConnectionPool(host='www.nationstrust.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nationstrust.com', port=443): Read timed out. (read timeout=2)"
natural-soap-directory.org,"HTTPSConnectionPool(host='natural-soap-directory.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd7b9936460>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='natural-soap-directory.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7b097e220>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
naturum.ne.jp,"HTTPSConnectionPool(host='naturum.ne.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='naturum.ne.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
